i18n/en.json

```diff
@@ -14,8 +14,16 @@
   "exportModalIncludeRelatedSettings": "Related Documents Settings",
   "exportModalNoRelatedTypes": "No Related Types",
   "import": "Import {{ type }}",
+  "importCleanFailed": "The cleaning of the imported file on server failed: {{ exportPath }}",
+  "importDuplicateContinue": "Continue Import",
+  "importDuplicateDetected": "Duplicates Detected.",
+  "importDuplicateMessage": "Check the items you'd like to overwrite during import.",
+  "imported": "Imported",
   "importFailed": "Import failed",
   "importFileError": "The {{ format }} file import failed",
+  "importFailedForSome": "{{ count }} documents encountered error during import",
+  "importing": "Importing {{ type }}...",
   "importModalDescription": "Importing content requires a {{ formats }} file. See our <a href=\"https://v3.docs.apostrophecms.org/\" target=\"_blank\">guide to importing content</a> in Apostrophe.",
```

Comment:
Should we remove the link or remember that we need to target the correct page?

Reply:
I removed it and said, official documention instead.